test/test-utils/test-utils.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+FAILED=()
```

Comment:
Worth explicitly declaring it as an array

https://google.github.io/styleguide/shellguide.html#arrays

```
declare -a FAILED
```